itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTablesBase.java

```diff
@@ -203,18 +203,17 @@ void verifyLoadExecution(String replicatedDbName, String lastReplId, boolean inc
 
   void prepareIncAcidData(String dbName) throws Throwable {
     primary.run("use " + dbName)
-            .run("create table t6 (str string) stored as orc tblproperties " +
-                    "(\"transactional\"=\"true\")")
-            .run("insert into t6 values ('aaa'), ('bbb')")
+            .run("create table t6 stored as orc tblproperties (\"transactional\"=\"true\")" +
+                    " as select * from t1")
```

Comment:
add a test for partitioned tables also

Reply:
https://issues.apache.org/jira/browse/HIVE-21678. Similar to above comment about testset changes.